### PR TITLE
removing dependency on /examples for ui integration tests, fix build

### PR DESCRIPTION
**Pull request by rns4731**

### CODE REVIEW

File: integration-testing/pom.xml [hunk 1]
```diff
@@ -31,6 +31,24 @@
...
+    <repositories>
+        <repository>
+            <id>nexus</id>
+            <url>https://repository.continuuity.com/content/groups/public</url>
+        </repository>
+        <repository>
+            <id>releases</id>
+            <url>https://repository.continuuity.com/content/repositories/releases/</url>
+        </repository>
+        <repository>
```
Comment (albertshau) [inline integration-testing/pom.xml:39]: this repo is private and shouldn't be needed

File: integration-testing/README.md [hunk 2]
```diff
@@ -2,5 +2,18 @@ Selenium testing for Loom.
 
 Runs with the test environment of loom ui server.
 
-node server.js --env=test
```
Comment (albertshau) [inline integration-testing/README.md:5]: do these tests require node.js? If so we should put a note here.  Any browser dependencies?

File: integration-testing/pom.xml (hunk 1 above)
Comment (rns4731) [inline integration-testing/pom.xml:39]: done.

File: integration-testing/README.md (hunk 2 above)
Comment (rns4731) [inline integration-testing/README.md:5]: done.

**Pull request merged**